components-sdk/src/locales/es.json

```
@@ -0,0 +1,63 @@
+{
+  "components.content": "Contenido",
+  "components.container": "Contenedor",
+  "components.image": "Imagen",
+  "components.file": "Archivo",
+  "components.separator": "Separador",
+  "components.button": "Botón",
+  "components.button-link": "Botón de enlace",
+  "components.select-menu": "Menú de selección",
+  "components.thumbnail": "Miniatura",
+  "components.text-input": "Entrada de texto",
+  "button.add-component": "Añadir componente",
+  "button.add-inline": "Añadir componente debajo",
+  "button.add-content": "Añadir contenido",
+  "button.add-button": "Añadir botón",
+  "button.new-option": "Nueva opción",
+  "error": "Error:",
+  "actions.add-action": "Añadir acción",
+  "file.upload-file": "Subir archivo",
+  "separator.margin": "Haz clic para cambiar el margen",
+  "thumbnail.set-image-url": "Usar imagen de internet",
+  "thumbnail.image-url": "URL de la imagen, https://...",
+  "thumbnail.upload-image": "Subir imagen",
+  "thumbnail.add-description": "Añadir texto alternativo",
+  "thumbnail.change-description": "Cambiar texto alternativo",
+  "thumbnail.clear-description": "Borrar texto alternativo",
+  "thumbnail.set-spoiler": "Spoiler",
+  "thumbnail.remove-spoiler": "Spoiler",
+  "thumbnail.delete": "Eliminar",
+  "thumbnail.add-image": "Añadir imagen",
+  "alttext.edit": "(Editar)",
+  "string-select.global-settings": "Configuración global",
+  "string-select.invalid": "INVÁLIDO:",
+  "string-select.mark-enabled": "Marcar todo como activado",
+  "string-select.mark-disabled": "Marcar todo como desactivado",
+  "string-select.add-placeholder": "Añadir placeholder",
+  "string-select.change-placeholder": "Cambiar placeholder",
+  "string-select.clear-placeholder": "Borrar placeholder",
+  "string-select.set-minimum": "Establecer número mínimo de opciones seleccionadas",
+  "string-select.set-maximum": "Establecer número máximo de opciones seleccionadas",
```

```suggestion
  "string-select.set-minimum": "Establecer mínimo de opciones seleccionables",
  "string-select.set-maximum": "Establecer máximo de opciones seleccionables",
```

Just to shorten it a bit, and change `seleccionadas` to `seleccionables`